elements/pfe-navigation/src/pfe-navigation-item.js

```diff
@@ -246,14 +246,25 @@ class PfeNavigationItem extends PFElement {
       // Get the LightDOM trigger & tray content
       this.trigger = this.querySelector(`[slot="trigger"]`);
   
-      this.directLink = this.trigger.querySelector("a");
-      this.linkUrl = this.directLink ? this.directLink.href : "#";
-  
-      // Turn off the fallback link
-      if (this.directLink) this.directLink.setAttribute("tabindex", "-1");
-  
-      this._trigger.addEventListener("click", this._navigateToUrl);
-      this._trigger.addEventListener("keyup", this._directLinkHandler);
+      // Check the light dom for the link
+      if (this.trigger) {
+        this.directLink = this.trigger.querySelector("a");
+        // Check the slotted content for the link if it can't be found
+        if (!this.directLink && this.trigger.tagName === "SLOT") {
+          let slottedContent = this.trigger.assignedNodes();
+          if (slottedContent.length > 0 && slottedContent[0].tagName === "A") {
+            this.directLink = slottedContent[0];
+          }
+        }
+
+        this.linkUrl = this.directLink ? this.directLink.href : "#";
+    
+        // Turn off the fallback link if the tray does not exist
+        if (this.directLink) this.directLink.setAttribute("tabindex", "-1");
```

Comment:
Should this have curly braces? I know it's not needed in a one-line if statement.

Reply:
@chrisdo1 This isn't my favorite style. Can you add curly braces and bump the statement inside those braces?

Reply:
Sure, I can do that.  Can I ask why it seems less clear?  I feel like it still reads very cleanly as english.

Reply:
There's also a lot of precedence in this particular file of the one-line if statement